components/user/login.js

```diff
@@ -148,24 +159,68 @@ function Login({ handleLogPageToggle }) {
               placeholder="Email"
               onChange={handleChange}
               value={data.email}
-              className=" tw-px-2 tw-border-b-[1px] tw-border-b-black tw-py-3 "
+              className="tw-px-2 tw-border-b-[1px] tw-border-b-black tw-py-3 "
             />
           </div>
-          <div className="tw-flex tw-flex-col tw-mb-10">
-            <label for="name" className="tw-text-3xl tw-font-medium ">
+
+          <div className="tw-flex tw-flex-col tw-mb-10 tw-relative">
+            <label for="name" className="tw-text-3xl tw-text-left tw-font-medium tw-mr-10"
+            >
               Password
             </label>
             <input
-              type="password"
+              type={isPasswordVisible ? 'text' : 'password'}
               name="password"
               placeholder="Password"
               onChange={handleChange}
               value={data.password}
-              className="tw-px-2 tw-border-b-[1px] tw-border-b-black tw-py-3 "
+              className="tw-px-2 tw-border-b-[1px] tw-border-b-black tw-py-3 tw-pr-16"
             />
+            <div
+              className="tw-absolute tw-inset-y-0 tw-right-0 tw-flex tw-items-center tw-px-4 tw-text-gray-600 tw-top-12"
+              onClick={togglePasswordVisibility}
+            >
+              {isPasswordVisible ? (
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  strokeWidth={1.5}
+                  stroke="currentColor"
+                  className="tw-h-8 tw-w-8"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M3.98 8.223A10.477 10.477 0 001.934 12C3.226 16.338 7.244 19.5 12 19.5c.993 0 1.953-.138 2.863-.395M6.228 6.228A10.45 10.45 0 0112 4.5c4.756 0 8.773 3.162 10.065 7.498a10.523 10.523 0 01-4.293 5.774M6.228 6.228L3 3m3.228 3.228l3.65 3.65m7.894 7.894L21 21m-3.228-3.228l-3.65-3.65m0 0a3 3 0 10-4.243-4.243m4.242 4.242L9.88 9.88"
+                  />
+                </svg>
```

Comment:
can this be stored in `asset` folder so this can be used anywhere as in your case you replicate the same code for the registration page.

Reply:
Hey, is it oke 
Visibillity.jsx and VisibillityOff.jsx
![image](https://github.com/anmode/grabtern-frontend/assets/94003231/89155c87-9e2e-49ad-8357-52876c0be5ab)

Reply:
sounds good!!